### PR TITLE
Add Nextstrain functionality

### PR DESCRIPTION
**Pull request by JenkeScheen**
With CLI endpoint so that we can download gene mutation data from https://nextstrain.org/pathogens#list

### CODE REVIEW

**Comment by JenkeScheen** [comment]
@hmacdope this is now working for all viruses/genes that we have available in https://github.com/asapdiscovery/choppa/blob/nextstrain/choppa/data/metadata/nextstrain.json. There are a few viruses that we'd _like_ to include (MERS-CoV, WNV) but their NextStrain entries are too borked (gene nomenclature is way off). 

Can you check why CI is failing here and review plz?

File: choppa/nextstrain.py [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+        if response.status_code == 404:
+            return None
+        else:
+            raise http_err
+
+
+def nextstrain_json_to_tree(json_dict, root=True, parent_cumulative_branch_length=None):
+    """
+    Follow the exact same logic that the NextStrain backend follows when calculating per-residue mutation frequencies.
+    By John Huddleston.
```
Comment (hmacdope) [inline choppa/nextstrain.py:151]: Do we have a link to the source code?

File: devtools/conda-envs/choppa.yaml [hunk 2]
```diff
@@ -32,9 +33,12 @@ dependencies:
...
   # docs
   - sphinx
-  - sphinx_rtd_theme 
+  - sphinx_rtd_theme
+
+  # nextstrain
+  - altair
+  - augur
   
   # Pip-only installs
```
Comment (hmacdope) [inline devtools/conda-envs/choppa.yaml:42]: remove junk below this

**Review by hmacdope — CHANGES_REQUESTED** [review]
Looks great, super minor stuff. I will fix tests

File: choppa/nextstrain.py (hunk 1 above)
Comment (JenkeScheen) [reply choppa/nextstrain.py:152]: here: https://github.com/nextstrain/auspice/blob/e6e4d1a67be4648839f5aff43cb13ce30b23aaa6/src/util/entropy.js#L45-L134, I'll insert.

**Review by hmacdope — APPROVED** [review]
LGTM, thanks for making the changes @JenkeScheen

**Pull request merged**